pydriller/metrics/process/change_set.py

```diff
@@ -36,6 +36,9 @@ def max(self):
 
         :return: int max number of files committed together
         """
+        if not self.committed_together:
+            return 0
+
         return max(self.committed_together)
```

Comment:
Nit: you can use `max(..., default=0)` instead

Reply:
Done, thanks!

Reply:
Yes nice catch!